### PR TITLE
Set up Appraisal and Travis testing

### LINKED ISSUE #14: Appraisal against latest Ruby versions (2.1, 2.2)

### PR DESCRIPTION
**Pull request by mxie**
For Rails 4.0/4.1/4.2 and Ruby 2.1/2.2.

And a few other updates as I was trying to get `rake` to work.

Closes #14

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,6 +2,8 @@ source "http://rubygems.org"
```
Comment (joshuaclayton) [inline Gemfile]: @mxie can you update the source to the https version and rebuild appraisals?

File: Appraisals [hunk 2]
```diff
@@ -0,0 +1,11 @@
+appraise "rails4.0" do
```
Comment (joshuaclayton) [inline Appraisals]: should this also test the latest 3.2?

File: Gemfile (hunk 1 above)
Comment (mxie) [inline Gemfile]: Yup! Done in 9b3e2ec5e3b3ff6202f4e311d30757412ecb6b25.

File: Appraisals (hunk 2 above)
Comment (mxie) [inline Appraisals]: Hmm, according to #13, we have a Rails 4 dependency. We could easily change that though if we'd like. I think this also means that we'd have to change the name of the `config.serve_static_files` flag in the dummy app to the Rails 3-compatible one (`server_static_assets` - deprecated in 4). I'm good either way.

Comment (mxie) [inline Appraisals]: Any further thoughts on this?

**Comment by joshuaclayton** [comment]
Given [JackUp has been deprecated](https://github.com/thoughtbot/jack_up/commit/89c9b1ec12e99c1345f09f233c32865b24696d11), I'm closing this.

**Pull request closed without merging**